contracts/src/v0.8/vrf/dev/VRFCoordinatorV2_5.sol

```diff
@@ -504,6 +505,11 @@ contract VRFCoordinatorV2_5 is VRF, SubscriptionAPI, IVRFCoordinatorV2Plus {
     bool nativePayment,
     bool onlyPremium
   ) internal view returns (uint96) {
+    // fulfilRandomWords msg.data has 772 bytes and with an additional
+    // buffer of 32 bytes, we get 804 bytes.
+    if (msg.data.length > 804) {
```

Comment:
should we move this check to fulfillRandomWords() function instead of the internal `_calculatePaymentAmount()` ?

Reply:
Hmm, sure